src/psyclone/psyGen.py

```diff
@@ -5576,6 +5571,7 @@ def _if_stmt_handler(self, node, parent):
         :rtype: :py:class:`psyclone.psyGen.IfBlock`
         '''
         ifblock = IfBlock(parent=parent, annotation='was_single_stmt')
+        ifblock.ast = node
```

Comment:
I presume this is a bug fix.

Reply:
Yes - we weren't setting the `._ast` member of IfBlock previously and that causes problems when walking the tree (in transformations).